packages/components/psammead-oembed/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@bbc/psammead-oembed",
+  "version": "0.1.0-alpha.1",
+  "publishConfig": {
+    "tag": "alpha"
+  },
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
+  "description": "A component that sanitizes and renders the HTML from an oEmbed response.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-oembed"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-oembed/README.md",
+  "dependencies": {
+    "dompurify": "^2.0.8"
```

Comment:
Dompurify as a package isn't that large (which is nice). We would just want to ensure that when we include this package in Simorgh that it is only loaded when we actually render oembed content in a effort to keep the page weight down.

I also think this should be a temp measure and that we should look to move the sanitisation to a backend service sometime soon.